src/store/store.ts

```diff
@@ -52,34 +57,39 @@ export const loading = (loadingStatus: boolean): LoadingAction => (
   }
 );
 
-export const setState = (todosList: TodoWithUser[], loadingStatus: boolean): SetStateAction => (
-  {
-    type: SET_STATE,
-    todosList,
-    loadingStatus,
-  }
-);
-
-export const alphabeticallyName = (alphabeticallyNameStatus: boolean): AlphabeticallyNameAction => (
-  {
-    type: CHANGE_ALPHABETICALLY_NAME,
-    alphabeticallyNameStatus,
-  }
-);
+export const setState
+  = (todosList: TodoWithUser[], loadingStatus: boolean): SetStateAction => (
+    {
+      type: SET_STATE,
+      todosList,
+      loadingStatus,
+    }
+  );
 
-export const alphabeticallyTitle = (alphabeticallyTitleStatus: boolean): AlphabeticallyTitleAction => (
-  {
-    type: CHANGE_ALPHABETICALLY_TITLE,
-    alphabeticallyTitleStatus,
-  }
-);
-
-export const completedTodo = (completedTodoStatus: boolean): CompletedTodoAction => (
-  {
-    type: CHANGE_COMPLETED,
-    completedTodoStatus,
-  }
-);
+export const sortedName
```

Comment:
this is a name for some value `name`, which was sorted, not for function that DO something => VERB name